openshift/_nginx.dc.json

```diff
@@ -181,12 +204,26 @@
                     "memory": "${MEMORY_REQUEST}"
                   }
                 },
+                "volumeMounts": [
+                  {
+                    "mountPath": "/var/log/nginx",
+                    "name": "${NAME}${SUFFIX}-log"
```

Comment:
Will there be one log PVC for all logs (nginx, flask, and postgres)? or one for each? Just wondering about folder/PVC naming.

Reply:
When @itchison and I chatted, we figured twas probably better to keep a single PVC to share between environments, that only holds logs, than assigning logs to every single environment.
Mostly to keep our PVC #'s down and if we lose the PVC there is no real harm, given that filebeat will immediately send it to elastic